submissions/tavor118sn/task_js_post_oop/index.js

```diff
@@ -0,0 +1,83 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+class Creature {
+  constructor(species, name, age, gender, legs, say, friends) {
+    this.species = species;
+    this.name = name;
+    this.age = age;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = say;
+    this._friends = friends
+  }
+
+  get friends() {
+    return (this._friends && this._friends.length) ? this._friends.join(', ') : '';
+  }
+
+  getAllCharacteristics() {
+    return [
+      `${this.species}`,
+      `<strong>${this.name}</strong>`,
+      `${this.age}`,
+      `${this.gender}`,
+      `${this.legs}`,
+      `${this.saying}`,
+      `${this.friends}`,
```

Comment:
Just wondering where this type of reference to variables (like `` `${varName}` ``) comes from? Please explain.
(`` `<strong>${this.name}</strong>` `` is out of the question)

Reply:
Sorry! Copy-pasted from my previous task solution. Will be fixed.

Reply:
@tavor118sn doesn't really answer the question.
The question was not "where did you take this fragment of code from" but "where this **type of referencing** comes from".

Reply:
I see. We have already discussed this topic in one of your past tasks.
See, how anti-patterns draw attention.